test/swoosh/adapters/postmark_test.exs

```diff
@@ -63,51 +69,95 @@ defmodule Swoosh.Adapters.PostmarkTest do
       |> text_body("Hello")
 
     Bypass.expect bypass, fn conn ->
-      conn = parse(conn)
-      body_params = %{"Subject" => "Hello, Avengers!",
-                      "To" => "\"Steve Rogers\" <steve.rogers@example.com>,wasp.avengers@example.com",
-                      "From" => "tony.stark@example.com",
-                      "Cc" => "thor.odinson@example.com,\"Bruce Banner\" <hulk.smash@example.com>",
-                      "Bcc" => "beast.avengers@example.com,\"Clinton Francis Barton\" <hawk.eye@example.com>",
-                      "ReplyTo" => "iron.stark@example.com",
-                      "TextBody" => "Hello",
-                      "HtmlBody" => "<h1>Hello</h1>"}
+      conn        = parse(conn)
+      body_params = %{
+        "Subject"  => "Hello, Avengers!",
+        "To"       => "\"Steve Rogers\" <steve.rogers@example.com>,wasp.avengers@example.com",
+        "From"     => "tony.stark@example.com",
+        "Cc"       => "thor.odinson@example.com,\"Bruce Banner\" <hulk.smash@example.com>",
+        "Bcc"      => "beast.avengers@example.com,\"Clinton Francis Barton\" <hawk.eye@example.com>",
+        "ReplyTo"  => "iron.stark@example.com",
+        "TextBody" => "Hello",
+        "HtmlBody" => "<h1>Hello</h1>"
+      }
 
       assert body_params == conn.body_params
-      assert "/email" == conn.request_path
-      assert "POST" == conn.method
+      assert "/email"    == conn.request_path
+      assert "POST"      == conn.method
```

Comment:
All the above: Style change, should be in a separate PR and across all adapters.